test/benchmark/discr/discr_mappings_test.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2014, 2015 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+
+# Imports for common Python 2/3 codebase
+from __future__ import print_function, division, absolute_import
+from future import standard_library
+standard_library.install_aliases()
+
+# External
+import pytest
+import numpy as np
+
+# Internal
+import odl
+from odl.discr.grid import sparse_meshgrid
+from odl.discr.discr_mappings import (
+    GridCollocation, NearestInterpolation, LinearInterpolation,
+    PerAxisInterpolation)
+
+
+pytestmark = pytest.mark.benchmark
+
+
+def initialize():
+    global shape, values, hcube, grid, space, dspace32, dspace64
+    global points, mesh
+    shape = (100, 100, 100, 100)
+    values = np.zeros(np.prod(shape))
+    hcube = odl.IntervalProd([0] * 4, [1] * 4)
+    grid = odl.uniform_sampling(hcube, shape, as_midp=True)
+    space = odl.FunctionSpace(hcube)
+    dspace32 = odl.Rn(grid.ntotal, dtype='float32')
+    dspace64 = odl.Rn(grid.ntotal, dtype='float64')
+    points = np.random.rand(4, 60 ** 4)
```

Comment:
Why not? 60 is a good number in terms of prime number decomposition. If we want an example with `ndim=5` and `ndim=6` with the same number of nodes, it's easily possible.
And, btw, this does not in any way have to do with the 100 in `shape`.